calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx

```diff
@@ -0,0 +1,32 @@
+xx July 2023
```

Comment:
I assume once you rebase this PR on latest master I expect this change will go away.